test/fuzz/gen.ml

```diff
@@ -298,33 +381,46 @@ let global env =
 
 let local = B.param
 
-let func env =
+let func env fuzz_conf =
+  let open Crowbar.Syntax in
   let* () = const () in
   Env.reset_locals env;
   Env.refill_fuel env;
   let* locals = list (local env) in
   let* type_f = B.block_type env in
   let (_name : string) = Env.add_block env type_f Env.Func in
-  let+ body = expr ~block_type:type_f ~stack:[] ~locals env in
+  let+ body =
+    Option.get @@ run (expr ~block_type:type_f ~stack:[] ~locals env) fuzz_conf
+  in
   Env.remove_block env;
   let id = Some (Env.add_func env type_f) in
   MFunc { type_f; locals; body; id }
 
-let fields env =
-  let* memory = option (memory env) in
+let fields env fuzz_conf =
+  let open Crowbar.Syntax in
+  let* memory =
+    match fuzz_conf with
+    | Concrete -> option (memory env)
+    | Symbolic -> const None
+  in
   let* datas = list (data env) in
   let* types = list (typ env) in
-  let* tables = list (table env) in
+  let* tables =
+    match fuzz_conf with Concrete -> list (table env) | Symbolic -> const []
```

Comment:
Why are we not creating tables in the symbolic case ? Too much stuff is unimplemented ?

Reply:
I created #190.